### PR TITLE
Reduce Jenkinsfile 

### PR DESCRIPTION
**Pull request by F-WRunTime**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,221 +0,0 @@
...
-                  mv web/public_content ./
-                  rm -rf $(find . -maxdepth 1 -not -name public_content -a -not -name .git -a -not -path . -a -not -path .. -a -not -name CNAME)
-                  mv public_content/* ./
-                  rm -rf public_content
-                  git add ./
-                  git commit -m 'gh-pages: Updated the website'
-                  git merge --strategy ours origin/gh-pages --allow-unrelated-histories
-                  git push origin gh-pages
-                '''
-              }
```
Comment (ehildenb) [inline Jenkinsfile:214]: This stage needs to be ported to GH Runners.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,221 +0,0 @@
...
-          }
-        }
-        stage('Update Dependents') {
-          steps {
-            build job: 'DevOps/master', propagate: false, wait: false                                                     \
-                , parameters: [ booleanParam ( name: 'UPDATE_DEPS'         , value: true                       )          \
-                              , string       ( name: 'UPDATE_DEPS_REPO'    , value: 'runtimeverification/evm-semantics' ) \
-                              , string       ( name: 'UPDATE_DEPS_VERSION' , value: "${env.LONG_REV}")                    \
-                              ]
-          }
```
Comment (ehildenb) [inline Jenkinsfile:189]: This need sto be ported to GH runners.

File: Jenkinsfile [hunk 3]
```diff
@@ -1,221 +0,0 @@
...
-                focal_name=kevm_${VERSION}_amd64_focal.deb
-                mv ../focal/kevm_${VERSION}_amd64.deb  ${focal_name}
-
-                echo "KEVM Release ${KEVM_RELEASE_TAG}"  > release.md
-                echo ''                                 >> release.md
-                cat INSTALL.md                          >> release.md
-                hub release create                                          \
-                    --attach ${focal_name}'#Ubuntu Focal (20.04) Package'   \
-                    --file release.md "${KEVM_RELEASE_TAG}"
-              '''
```
Comment (ehildenb) [inline Jenkinsfile:178]: This needs to be ported to GH Runners.

**Comment by F-WRunTime** [comment]
Decided to leave some components in Jenkinsfile for now

**Pull request closed without merging**